ingeniamotion/configuration.py

```diff
@@ -876,6 +929,43 @@ def restore_tcp_ip_parameters(self, servo: str = DEFAULT_SERVO) -> None:
             raise IMException("TCP IP parameters can only be restored in ethernet servos.")
         drive.restore_tcp_ip_parameters()
 
+    def get_mac_address(
+        self, servo: str = DEFAULT_SERVO, string_format: bool = False
+    ) -> Union[int, str]:
```

Comment:
You could scope the return type with overloads: https://mypy.readthedocs.io/en/stable/more_types.html#function-overloading